splash/lua_runner.py

```diff
@@ -90,31 +87,51 @@ def on_lua_error(self, lua_exception):
 
     def dispatch(self, cmd_id, *args):
         """ Execute the script """
-        args = args or None
-        args_repr = truncated("{!r}".format(args), max_length=400, msg="...[long arguments truncated]")
-        self.log("[lua_runner] dispatch cmd_id={}, args={}".format(cmd_id, args_repr))
+        args = args or ()
+
+        def truncated_repr(x):
```

Comment:
This can be used in a few other places - what do you think about moving this function to utils?